app/controllers/concerns/batch_processable.rb

```diff
@@ -8,12 +8,12 @@ def restrict_scope_to_selected_objects(rel)
     rel.where(id: params[:selected].keys)
   end
 
-  def restrict_by_search_and_ability_and_selection(scope, klass, search_options: {})
+  def restrict_by_search_and_ability_and_selection(relation)
```

Comment:
Renamed `scope` => `relation` because it seems like that's more accurate, please correct if I'm wrong.

Reply:
Yes if that gets passed through to the relation param, should be good. `scope` and `relation` are used interchangeably in other places which is probably why this happened.